### ISSUE TITLE
Missing Title for Delete annotation button (Trash icon)

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2698]
Other buttons in this bar have title argument. It is missing for the trash icon.

![image](https://github.com/user-attachments/assets/4c6b9ad2-c8b6-44f0-aae8-5957b1f9de9d)
